### PR TITLE
Support latest API of Unique Building ID

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
There was a small change to the API in the UBID API.

#### What's this PR do?
Enables use of new UBID API. Note that UBID-V2 is no longer supported, only V3.

#### How should this be manually tested?
The unit tests should cover the tests. Otherwise, import one of the covered building sample files and verify that the UBID imports and that the GeoJSON exports look correct in geojson.io.

#### What are the relevant tickets?
#1930

### CODE REVIEW

**Comment by RDmitchell** [comment]
@nllong -- does this mean that the UBIDs generated for San Jose are not valid, and need to be regenerated, or is it just the interpretation of the UBIDs that changed?

**Comment by nllong** [comment]
Nope. Everything is still valid. We were using V3 already. We just updated
the dependent library.

Il giorno mar 6 ago 2019 alle 14:43 RDmitchell <notifications@github.com>
ha scritto:

File: seed/tests/test_utils_ubid.py [hunk 1]
```diff
@@ -162,20 +162,23 @@ def test_decode_ulids_does_nothing_if_no_ULID_provided(self):
     def test_decode_ubids_is_successful_when_v2_format_C_NW_SE_UBID_provided(self):
         property_details = self.property_state_factory.get_details()
         property_details['organization_id'] = self.org.id
-        property_details['ubid'] = '849VQJH6+95J-849VQJH5+VGW-849VQJG6+XV8'
+        # Old format (v2) is not longer supported
```
Comment (adrian-lara) [inline seed/tests/test_utils_ubid.py]: v3 is tested already, so this particular test could be removed. Not a big deal either way though.

**Comment by nllong** [comment]
cool thanks. I'll remove the unneeded test.

**Comment by adrian-lara** [comment]
> [Coverage Status](https://coveralls.io/builds/25046243)
> 
> Coverage decreased (-0.002%) to 74.468% when pulling **[25b2ab9](https://github.com/SEED-platform/seed/commit/25b2ab939dd602eeb8421d8839d37dcf1374f894) on upgrade-ubid** into **[8ba06bf](https://github.com/SEED-platform/seed/commit/8ba06bf794699a3170d3e1e1fa22e4bbe12ccaaa) on develop**.

This isn't what I expected... Looking into it now

**Comment by adrian-lara** [comment]
> > [Coverage Status](https://coveralls.io/builds/25046243)
> > Coverage decreased (-0.002%) to 74.468% when pulling **[25b2ab9](https://github.com/SEED-platform/seed/commit/25b2ab939dd602eeb8421d8839d37dcf1374f894) on upgrade-ubid** into **[8ba06bf](https://github.com/SEED-platform/seed/commit/8ba06bf794699a3170d3e1e1fa22e4bbe12ccaaa) on develop**.
> 
> This isn't what I expected... Looking into it now

Never mind. Looks like it was just because there was still 1 red line among less green lines. 
Still looks good to me 👍

**Pull request merged**